### DISCUSSION TITLE
How can I hide other windows and restore them back when I wan to focus on one specific window?

### DISCUSSION THREAD

**Discussion by mrwensim** [asmvik/yabai#1534] [Q&A]
Hi folks, when I try to focus on one window from a specific application (with transparent background) with zoom-fullscreen, is it possible to auto hide all the other windows, and restore them back when I toggle zoom-fullscreent for this specific application?

**Comment by jrock2004** [comment]
Did you find a solution?

**Comment by Bellavene** [comment]
I believe you can make of use of jankyborder in such case.
It can render a colored backdrop for a window, so you can script a fullscreen toggle for that.

**Comment by Straven** [comment]
I used a simple workaround. If I'm focusing on transparent app I make all other apps fully transparent. That simple trick works fine.
```
# Get focused window app name
focused_app=$(yabai -m query --windows --window | jq -r '.app')

# Check if app is in transparent list
if echo "$focused_app" | grep -qE "$transparent_apps"; then
  # Transparent app focused
  yabai -m config active_window_opacity "$opacity_value"
  yabai -m config normal_window_opacity 0.00001
else
  # Non-transparent app focused
  yabai -m config active_window_opacity 1.0
  yabai -m config normal_window_opacity 1.0
fi
```
